src/main/java/org/folio/consortia/service/impl/UserTenantServiceImpl.java

```diff
@@ -123,10 +122,10 @@ public UserTenant save(UUID consortiumId, UserTenant userTenantDto, boolean isSy
     if (isSystemUserContextRequired) {
       createOrUpdateShadowUserWithSystemUserContext(userTenantDto.getUserId(), shadowUser, userTenantDto);
     } else {
-      createOrUpdateShadowUserWithRequestedContext(userTenantDto.getUserId(), shadowUser, userTenantDto, currentTenantContext);
+      createOrUpdateShadowUserWithRequestedContext(userTenantDto.getUserId(), shadowUser, userTenantDto, folioExecutionContext);
     }
 
-    try (var context = new FolioExecutionContextSetter(prepareContextForTenant(currentTenantId, folioModuleMetadata, currentTenantContext))) {
+    try (var context = new FolioExecutionContextSetter(prepareContextForTenant(currentTenantId, folioModuleMetadata, folioExecutionContext))) {
```

Comment:
It looks like setting of the context with same tenantId as it already has, isn't it? Is it needed?

Reply:
I checked, it turned out useless, Thanks to find